Makefile

```diff
@@ -5,7 +5,7 @@ include Makefile.include
 env-up: 							## Start devcontainer.
 	docker-compose up -d
 
-env-up-rebuild: env-update-image	## Start devcontainer with rebuild.
+env-up-rebuild:						## Rebuild and start devcontainer
```

Comment:
I think it's better to change the behavior of env-update-image to pull the latest image based on some environment variable and it should be the same as the one used in docker-compose.yml.
P.S. PMM_VERSION doesn't look like a good name for this environment variable :)

Reply:
I think it used to be `PMM_SERVER_IMAGE`, will update it to use that then.